components/Cards/examples/CardsCollectionTintingExample.m

```diff
@@ -22,7 +22,9 @@ @interface CardsCollectionTintingExample ()
 
 @end
 
-@implementation CardsCollectionTintingExample
+@implementation CardsCollectionTintingExample {
+  BOOL _interactable;
```

Comment:
Please switch to a property or [mark this as a `@private` iVar](http://google.github.io/styleguide/objcguide.html#instance-variables-in-headers-should-be-protected-or-private)